### PR TITLE
Open comments mechanism

### PR DESCRIPTION
**Pull request by rshulyar**

### CODE REVIEW

File: src/components/body/commentsWindow/commentsWindow.js [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+        }
+    }
+        
+
     return (
         <div className={classes['wrap']}>
             <div className={classes['title']}>Comments list</div>
             <div className={classes['comments-container']}>
-                <Comments/>
+                {addCommentsList()}
```
Comment (dobryanskyy) [inline src/components/body/commentsWindow/commentsWindow.js:30]:
```js
{storeSelectedItem && storeItemsBase[storeSelectedItem].comments.map((item, i) => <Comments
                key={i}
                index={i}
                text={item}
            />)}
```

File: src/reducers/itemsBase.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                ...state.slice(action.payload.indexItem + 1)]
+            );
+        }
+        case 'finishLoadItemsBase': {
+            return action.payload;
+        }
+        case 'updateState': {
+            const update = JSON.stringify(state);
+            localStorage.setItem('itemsBase', update);
+            console.log(state);
```
Comment (dobryanskyy) [inline src/reducers/itemsBase.js:45]: This code cannot be in reducer!
Reducer can ONLY modify state. This code should be moved to saga

File: src/reducers/selectedItem.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const initialState = 0;
+
+function selectedItem(state = initialState, action) {
+    switch (action.type) {
+        case 'selected': {
+            localStorage.setItem('selectedItem', action.payload);
+            return action.payload;
```
Comment (dobryanskyy) [inline src/reducers/selectedItem.js:7]: should be moved to saga

**Review by dobryanskyy — CHANGES_REQUESTED** [review]

File: src/reducers/itemsBase.js (hunk 2 above)
Comment (rshulyar) [reply src/reducers/itemsBase.js]: fixed

File: src/components/body/commentsWindow/commentsWindow.js (hunk 1 above)
Comment (rshulyar) [reply src/components/body/commentsWindow/commentsWindow.js]: fixed

File: src/reducers/selectedItem.js (hunk 3 above)
Comment (rshulyar) [reply src/reducers/selectedItem.js]: fixed

**Pull request merged**